ports/servoshell/desktop/app.rs

```diff
@@ -154,6 +157,22 @@ impl App {
         let servo = servo_builder.build();
         servo.setup_logging();
 
+        // Initialize WebDriver server here before `servo` is moved.
+        if let Some(port) = self.opts.webdriver_port {
```

Comment:
Then this can just be:

```suggestion
        let webdriver_receiver = self.opts.webdriver_port.map(|port| {
            let (embedder_sender, embedder_receiver) = unbounded();

            // TODO: WebDriver will no longer need this channel once all WebDriver
            // commands are executed via the Servo API.
            let constellation_sender_deprecated = servo.constellation_sender();
            webdriver_server::start_server(
                port,
                constellation_sender_deprecated,
                embedder_sender,
                self.waker.clone(),
            );
            
            embedder_receiver
        });
```